src/components/simple-material/simple-material.jsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import PropTypes from "prop-types";
+import urlPropType from "url-prop-type";
+import replacePlaceholders from "../../core/replacePlaceholders";
+import Skeleton from "../atoms/skeleton/skeleton";
+
+export function SimpleMaterialSkeleton({ style }) {
+  return (
+    <div className="ddb-simple-material-skeleton" style={style}>
+      <figure className="ddb-reset">
+        <Skeleton br="0px" mb="0px" mt="0px" height="154px" width="100px" />
+      </figure>
+      <div>
+        <Skeleton width="45px" mb="12px" />
+        <Skeleton width="145px" mb="12px" />
+        <Skeleton width="95px" />
+      </div>
+    </div>
+  );
+}
+
+SimpleMaterialSkeleton.propTypes = {
+  style: PropTypes.objectOf(PropTypes.any)
+};
+
+SimpleMaterialSkeleton.defaultProps = {
+  style: {}
+};
+
+function SimpleMaterial({
+  item,
+  materialUrl,
+  authorUrl,
+  ofText,
+  className,
+  dataClass,
+  style
+}) {
+  return (
+    <section className={`ddb-simple-material ${className}`} style={style}>
+      {item.coverUrl && (
+        <figure className="ddb-simple-material__cover">
+          <a
+            href={replacePlaceholders({
+              text: materialUrl,
+              placeholders: {
+                pid: encodeURIComponent(item.pid)
+              }
+            })}
+          >
+            <img src={item.coverUrl} alt={item.title} />
+          </a>
+        </figure>
+      )}
+      <div className={`ddb-simple-material__data ${dataClass}`}>
+        {item.type}
+        <a
+          href={replacePlaceholders({
+            text: materialUrl,
+            placeholders: {
+              pid: encodeURIComponent(item.pid)
+            }
+          })}
+        >
+          <h2>{item.title}</h2>
+        </a>
+        <p className="ddb-simple-material__author-year">
+          {item.creators && (
+            <>
+              <span>{`${ofText} `}</span>
+              {item.creators.map((creator, index) => {
+                return (
+                  <span key={creator}>
+                    <a
+                      href={replacePlaceholders({
+                        text: authorUrl,
+                        placeholders: {
+                          author: encodeURIComponent(creator)
+                        }
+                      })}
+                    >
+                      {creator}
+                    </a>
+                    {item.creators[index + 1] ? ", " : " "}
+                  </span>
+                );
+              })}
+            </>
+          )}
+          {item.year && `(${item.year})`}
```

Comment:
```suggestion
          {item.year && `<time datetime="${item.year}" className="ddb-simple-material__year">${item.year}</time>`}
```

If we drop the `()` we can add them using pseudo elements in CSS instead.

Reply:
I'l rather add the brackets in the html itself instead of hard to mannage pseudo elements. :+1:

Reply:
The above was all weird..